arviz/data/base.py

```diff
@@ -237,7 +237,7 @@ def make_attrs(attrs=None, library=None):
     return default_attrs
 
 
-def _extend_xr_method(func):
+def _extend_xr_method(func, doc=None, description=None, examples=None, see_also=None):
```

Comment:
strings are not mutable, so there is no need to do the `None` trick like with dicts or lists. You can directy set the default to `doc="",`...